README.md

```diff
@@ -1 +1,55 @@
-# Conference-page
\ No newline at end of file
+## Welcome 
+
+
+This is a html page which is contains all information about World Games 2022 event I have used HTML and CSS and javascript for this project and you can use it in your local machin just by cloning this repo.
+
+## 👁 Video discribtion
+[How it works](https://www.loom.com/share/3cc4cc3bc8494798b7878b7cc2e5a09c)
+
+## 🖥️ Live Demo
+You can use this link [World Games 2022](https://zakiyousufi.github.io/Conference-page/) visiting this website
```

Comment:
Your deployment seems to be off, it's leading me to a copy of your README, kindly fix this issue :monocle_face: